webpack/assets/javascripts/react_app/routes/Statistics/constants.js

```diff
@@ -0,0 +1,9 @@
+import { foremanUrl } from '../../../foreman_tools';
+
+export const STATISTICS_PAGE_FETCH_META = 'STATISTICS_PAGE_FETCH_META';
+export const STATISTICS_PAGE_HIDE_LOADING = 'STATISTICS_PAGE_HIDE_LOADING';
+export const STATISTICS_PAGE_SHOW_MESSAGE = 'STATISTICS_PAGE_SHOW_MESSAGE';
+
+export const STATISTICS_PAGE_URL = foremanUrl
+  ? foremanUrl('/statistics')
+  : '/statistics';
```

Comment:
imho this is not required, the function will return the right value

Reply:
@sharvit `foremanUrl` is undefined in tests for some reason. any idea?

Reply:
is this still an issue with the mock?

Reply:
yes, `foremanUrl` is undefined in tests.

Reply:
hmm.. any idea why? @sharvit @amirfefer ?